src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -48,21 +54,31 @@ const App = () => (
 );
 
 const MyChild = () => {
-  const { tileId, isVideoEnabled, setIsVideoEnabled, toggleVideo } = useLocalVideo();
+  const { tileId, isVideoEnabled, setIsVideoEnabled, 
+  isReachVideoLimit, setIsReachVideoLimit, toggleVideo } = useLocalVideo();
 
   return (
     <p>Tile ID: {tileId}</p>
 
     <p>
-      {setIsVideoEnabled ? 'LocalVideo is enabled' : 'LocalVideo is disabled'}
+      {isVideoEnabled ? 'LocalVideo is enabled' : 'LocalVideo is disabled'}
+    </p>
+
+    <p>
+      {isReachVideoLimit ? 'Video limit is reached' : 'Video limit is not reached'}
```

Comment:
There's no need for the "is". 'Video limit reached' or 'Video limit not reached' is correct grammer

Reply:
Thanks, updated